pkg/user/appointee/types.go

```diff
@@ -0,0 +1,18 @@
+package appointee
+
+import (
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"math/big"
+)
+
+type CanCallConfig struct {
```

Comment:
technically this can be private `canCallConfig`
I know the other command has it exported but I think we don't need it to be exported as we don't use it outside the packages